.github/scripts/update_version.py

```
@@ -0,0 +1,102 @@
+    """Write updated content back to version file."""
+    file_path.write_text(content)
+
+
+def main() -> None:
+    """Main entry point for version update script."""
+    # Get repository root (script is in .github/scripts/)
+    repo_root = Path(__file__).parent.parent.parent
+    version_file = repo_root / "fooocus_version.py"
+
+    if not version_file.exists():
+        raise FileNotFoundError(f"Version file not found: {version_file}")
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)